python/arcticdb/version_store/library.py

```diff
@@ -201,6 +201,24 @@ class ReadRequest(NamedTuple):
     query_builder: Optional[QueryBuilder] = None
 
 
+class ReadInfoRequest(NamedTuple):
```

Comment:
Could you check whether this will show up in the docs like the other one? https://docs.arcticdb.io/api/library/arcticdb.version_store.library.ReadRequest

Reply:
ok, added the corresponding index in library.rst as discussed